x-pack/solutions/observability/plugins/apm/server/assistant_functions/get_apm_downstream_dependencies.ts

```diff
@@ -61,11 +61,13 @@ export function registerGetApmDownstreamDependenciesFunction({
     },
     async ({ arguments: args }, signal) => {
       return {
-        content: await getAssistantDownstreamDependencies({
-          arguments: args,
-          apmEventClient,
-          randomSampler,
-        }),
+        content: {
+          dependencies: await getAssistantDownstreamDependencies({
```

Comment:
@SrdjanLL Can we enforce this at compile time via types? Currently `content: any`:

https://github.com/elastic/kibana/blob/2e6f75380fee2132f99604211fc5a002ee90fbd2/x-pack/platform/plugins/shared/observability_ai_assistant/common/functions/types.ts#L31